packages/central-server/app/sync/CentralSyncManager.js

```diff
@@ -262,13 +265,22 @@ export class CentralSyncManager {
       await repeatableReadTransaction(store.sequelize, async (transaction) => {
         // do not need to update pending records when it is initial build
         // because it uses ticks from the actual tables for updated_at_sync_tick
-        if (!isInitialBuildOfLookupTable) {
+        if (isInitialBuildOfLookupTable) {
+          await this.store.models.SyncLookupTick.create({
+            sourceStartTick: previouslyUpToTick,
+            lookupEndTick: currentTick,
+          });
```

Comment:
@passcod can you make sure `updateLookupTable` has been called once before you start any data generation? Just in case it's an initial build difference catching some of the data in a way we haven't dealt with.

Reply:
I think the scheduler has it running every 20 seconds? If so it was definitely run. I can do extra ones manually tho

Reply:
You mean it definitely was run prior to starting any of the data generation right? Because you wouldn't have kicked off the data generation / testing stuff within 20 seconds of the deployment coming up anyway